Cloud Rack Average Energy Usage.json

```diff
@@ -0,0 +1,5313 @@
+{
+  "annotations": {
+    "list": [
+      {
+        "builtIn": 1,
+        "datasource": {
+          "type": "grafana",
+          "uid": "-- Grafana --"
+        },
+        "enable": true,
+        "hide": true,
+        "iconColor": "rgba(0, 211, 255, 1)",
+        "name": "Annotations & Alerts",
+        "type": "dashboard"
+      }
+    ]
+  },
+  "editable": true,
+  "fiscalYearStartMonth": 0,
+  "graphTooltip": 0,
+  "id": 12,
+  "links": [],
+  "liveNow": false,
+  "panels": [
+    {
+      "datasource": {
+        "type": "grafana-opensearch-datasource",
+        "uid": "a3bded11-df72-4bb7-9935-5fbbeca21a29"
```

Comment:
Would it be possible to replace the uid for the datasource `a3bded11-df72-4bb7-9935-5fbbeca21a29` throughout the JSON file with something like `opensearch-grafana`?  

This is just so that the uid for the datasource in the dashboard matches the uid we will give the datasource when we add the datasource into our provisioning file